packages/terra-paginator/src/ProgressivePaginator.jsx

```diff
@@ -47,6 +47,14 @@ const propTypes = {
    * Allows user to set custom page label. _(usage note: User must pass translated text)_.
    */
   pageLabel: PropTypes.string,
+  /**
+   * Apply fusion anchor style or not.
+   */
+  fusionAnchor: PropTypes.bool,
```

Comment:
Any consideration to auto apply this prop when theme.className is "orion-fusion-theme"? Do you think this prop is useful to have on its own even if current theme is not "orion-fusion-theme"?

Reply:
MPage PaginatorFooter has 2 styles: anchor(top borderline and gray background) and canvas(no borderline and no background color). My initial thoughts would be that since we need to support both and we don't know which one is preferred by the consumer, I would leave the consumer to set the prop depending on their usage. 

I feel like this prop would be specifically for MPage's usage since no other consumer teams request this requirement. The reason for adding a prop to control this style is that we need to support both styles within the fusion theme, instead of applying a new theme.